flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/nonha/embedded/EmbeddedHaServicesTest.java

```diff
@@ -57,7 +57,8 @@ public void setupTest() {
     @After
     public void teardownTest() throws Exception {
         if (embeddedHaServices != null) {
-            embeddedHaServices.closeAndCleanupAllData();
+            embeddedHaServices.cleanupAllData();
+            embeddedHaServices.close();
```

Comment:
```suggestion
            embeddedHaServices.closeAndOptionalCleanup(true);
```
nit: This is up to you. Calling the newly added default implementation (here and in the other tests) would make the diff smaller.